src/DiagnosticsDialog.cpp

```diff
@@ -51,6 +51,14 @@ DiagnosticsDialog::DiagnosticsDialog(QWidget* parent):
     dumpVersionInfo();
 }
 
+class Sleeper : public QThread
+{
+public:
+    static void usleep(unsigned long usecs){QThread::usleep(usecs);}
+    static void msleep(unsigned long msecs){QThread::msleep(msecs);}
+    static void sleep(unsigned long secs){QThread::sleep(secs);}
+};
```

Comment:
what is this abomination?!

Why not just use `QThread::sleep` directly? 

PS: interval is an int, times 1000 clearly means we want to sleep a certain number of seconds...